committee-steering/governance/sig-governance.md

```diff
@@ -1,60 +1,136 @@
+Within this section "Lead" refers to someone who is a member of the union
+ of a Chair, Tech Lead or Subproject Owner role. There is no one lead to any
+ Kubernetes community group. Leads have specific decision making power over some
+ part of a group and thus additional accountability. Each role is detailed below.  
 
-- Initial members are defined at the founding of the SIG or Subproject as part of the acceptance
-  of that SIG or Subproject.
-- Members *SHOULD* remain active and responsive in their Roles.
-- Members *MUST* be [community members] to be eligible to hold a leadership role
-  within a SIG.
-- Members taking an extended leave of 1 or more months *SHOULD*
-  coordinate with other members to ensure the
+- Initial roles are defined at the founding of the SIG or Subproject as part
+of the acceptance of that SIG or Subproject.
+
+#### Activity Expectations  
+- Leads *SHOULD* remain active and responsive in their Roles.
+- Leads taking an extended leave of 1 or more months *SHOULD*
+  coordinate with other leads to ensure the
   role is adequately staffed during the leave.
-- Members going on leave for 1-3 months *MAY* work with other
-  members to identify a temporary replacement.
-- Members of a role *SHOULD* remove any other members that have not communicated a
-  leave of absence and either cannot be reached for more than 1 month or are not
-  fulfilling their documented responsibilities for more than 1 month.
-  This may be done through a [super-majority] vote of members, or if there are not
-  enough *active* members to get a super-majority of votes cast, then removal may occur
-  through a [super-majority] vote between Chairs, Tech Leads and Subproject Owners.
-- Membership disagreements may be escalated to the SIG Chairs.  SIG Chair membership
-  disagreements may be escalated to the Steering Committee.
-- Members *MAY* decide to step down at anytime and propose a replacement.  Use lazy consensus amongst
-  other members with fallback on majority vote to accept proposal.  The candidate *SHOULD* be supported by a
-  majority of SIG Members or Subproject Contributors (as applicable).
-- Members *MAY* select additional members through a [super-majority] vote amongst members. This
-  *SHOULD* be supported by a majority of SIG Members or Subproject Contributors (as applicable).
+- Leads going on leave for 1-3 months *MAY* work with other
+  Leads to identify a temporary replacement.
+- Leads of a role *SHOULD* remove any other leads or roles that have not
+communicated a leave of absence and either cannot be reached for more than 1
+month or are not fulfilling their documented responsibilities for more than 1 month.
+  This may be done through a [super-majority] vote of Leads, or if there are
+  not enough *active* Leads to get a super-majority of votes cast, then
+  removal may occur through a [super-majority] vote between Chairs, Tech Leads
+  and Subproject Owners.
```

Comment:
This is confusing and may need an editorial followup.. if Leads are the union of Chairs+TL+SOs, then isn't this saying the same group twice?